schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INTEGER,
     neutered BOOLEAN,
-    weight_kg DECIMAL
+    weight_kg DECIMAL,
+    species VARCHAR
```

Comment:
- Hey, please, let's specify the numbers of characters the column can take to help maintain our database to improve its performance. Not specifying means the column can take any number of characters and this is not ideal for your database. eg `varchar(100);`

Reply:
#### README FILE

- Hey, great job adding the readme file, however, let's kindly update our readme to the current readme [template](https://github.com/microverseinc/readme-template) to conform to the new structure and be consistent to make our readme more professional.